addons/binding/org.openhab.binding.kodi/ESH-INF/thing/thing-types.xml

```diff
@@ -168,4 +175,16 @@
 		<description>Media type of the current file</description>
 		<state readOnly="true" pattern="%s"></state>
 	</channel-type>
+	<channel-type id="thumbnail" advanced="true">
+		<item-type>String</item-type>
```

Comment:
I already wanted to use the Image item type at first, but then I found this thread: https://community.openhab.org/t/displaying-an-image-item/22636
As I primary want to use the images in HABPanel, I switched back to plain String items.

So my suggestion would be: renaming these items appending `Url` and adding two Image items later as soon as [eclipse/smarthome#3448](https://github.com/eclipse/smarthome/issues/3448) is available. This would be the same workaround as in Onkyo Binding. What do you think?